zebra-chain/Cargo.toml

```diff
@@ -8,19 +8,26 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-
 default = []
 proptest-impl = ["proptest", "proptest-derive"]
 
 [dependencies]
+aes = "0.6"
 bech32 = "0.8.0"
+bitflags = "1.2.1"
 bitvec = "0.17.4"
 blake2b_simd = "0.5.11"
 blake2s_simd = "0.5.11"
 bs58 = { version = "0.4", features = ["check"] }
 byteorder = "1.4"
 chrono = { version = "0.4", features = ["serde"] }
+displaydoc = "0.2.0"
+fpe = "0.4"
+# Temporary workaround for https://github.com/myrrlyn/funty/issues/3
```

Comment:
Let's open a ticket so we remember to remove this workaround?

Reply:
Done: 2fcc368a
https://github.com/ZcashFoundation/zebra/issues/2082